tests/end_to_end/test_helper.cpp

```diff
@@ -718,6 +757,15 @@ _test_helper_end_to_end::~_test_helper_end_to_end()
         _unload_all_native_modules();
 
         clear_program_info_cache();
+
+        {
+            std::unique_lock lock(_overlapped_buffers_mutex);
+            if (!_overlapped_buffers.empty()) {
+                std::cerr << "Overlapped buffers not empty" << std::endl;
```

Comment:
Is this just for informational purposes?  (i.e., not an error?)  Maybe add a // comment

Reply:
This was part of debugging early on. Removed.